R/pkg/R/functions.R

```diff
@@ -1827,15 +1827,15 @@ setMethod("to_json", signature(x = "Column"),
 
 #' to_timestamp
 #'
-#' Converts the column into a TimestampType. You may optionally specify a format
+#' Converts the column into a timestamp column. You may optionally specify a format
```

Comment:
Is there more info on `timestamp column`? as it was, `TimestampType` we could say or reference the Scala/Spark type